import-service/handlers/import-products-file/import-products-file.spec.ts

```diff
@@ -0,0 +1,25 @@
+import * as AWSMock from "aws-sdk-mock";
+import {importProductsFile} from "./import-products-file";
+import {APIGatewayProxyEvent, Context} from "aws-lambda";
+
+describe("importProductsFile", () => {
+  beforeAll(async (done) => {
+    AWSMock.mock('S3', 'getSignedUrlPromise', (_action, _params) => {});
+    done();
+  });
+  const context = {} as unknown as Context;
+  const callback = () => {};
+  const event = {queryStringParameters: {csvFileName: 'csvFileName.csv'}} as unknown as APIGatewayProxyEvent;
+  it('should return status 200', async () => {
+    const res = await importProductsFile(event, context, callback) || {body: '', statusCode: null};
+    expect(res.statusCode).toBe(200);
```

Comment:
[minor] It would be useful to check also if `importProductsFile` returns correct signed url in body